modules/util/PrefixedLogger.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Class adds prefix to each log message.
+ */
+export default class PrefixedLogger {
+
+    /**
+     * Create new <tt>PrefixedLogger</tt>
+     * @param {Logger} logger
+     * @param {string} prefix string which wil be added at the beginning of each
+     * log message.
+     */
+    constructor(logger, prefix) {
+        this.logger = logger;
+        this.prefix = prefix;
+    }
+
+    /**
+     * Logs on TRACE logging level.
+     * @param text
+     * @param args
+     * @constructor
+     */
+    t(text, ...args) {
```

Comment:
(please don't hate me) Why don't you use the same names as Logger? That would make it easier to interchange.

Reply:
It's for the expression to be as short as possible, because wrapping template strings is PITA. Also note that it would not replace 'logger' which can be used from static methods.

Reply:
> t's for the expression to be as short as possible, because wrapping template strings is PITA.

Hum, `this.log.e('There is more than 1 video track !', localVideos);` is not that much of an improvement over `this.log.error('There is more than 1 video track !', localVideos);` IMHO, and removed the need for the reader to remember they are different.



> Also note that it would not replace 'logger' which can be used from static methods.

Sure, I mean their APIs are interchangeable (mostly).